thoth-core/src/main/java/fr/maif/eventsourcing/impl/InMemoryEventStore.java

```diff
@@ -118,9 +118,7 @@ public Materializer materializer() {
     @Override
     public Source<EventEnvelope<E, Meta, Context>, NotUsed> loadEventsByQuery(Query query) {
         return Source.from(eventStore)
-                .filter(e -> {
-                    return Option.of(query.entityId).map(id -> id.equals(e.entityId)).getOrElse(true);
-                });
+                .filter(e -> Option.of(query.entityId).map(id -> id.equals(e.entityId)).getOrElse(true));
```

Comment:
I think this modification is unecessary for this feature. Could you remove it to clarify ?